File: backend/Models/Household.cs
public class Household
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string? Code { get; set; }
    public ICollection<Profile> Profiles { get; set; } = new List<Profile>();
    public ICollection<HouseholdTask> Tasks { get; set; } = new List<HouseholdTask>();
}

File: types/Household.ts
import { HouseholdTask } from './HouseholdTask';
import { Profile } from './profile';

export type Household = {
  id: number;
  name: string;
  code: string;
  profiles: Profile[];
  tasks: HouseholdTask[];
};

export type CreateHousehold = {
  name: string;
};

export type joinHousehold = {
  code: string;
};
